private/bufpkg/bufconfig/input_config.go

```diff
@@ -182,7 +184,7 @@ func NewGitRepoInputConfig(
 		location:          location,
 		subDir:            subDir,
 		branch:            branch,
-		tag:               tag,
+		commitOrTag:       tag,
```

Comment:
I think the parameter should be renamed to `commitOrTag` in this case

Reply:
done